### ISSUE TITLE
The results of using seed label visualization when reproducing the paper are not quite accurate

### ISSUE THREAD

**Issue by Truring** [YurongYou/MODEST#9]
I used OpenPCDet's visualization method to visualize seed labels, but it didn't look quite right. The 3D boxes are floating on the ground and cannot properly wrap around obstacles. 
The point cloud I used is "*. bin" files in data_preprocessing/nuscenes/NUSCENESS_KITTI-FORMAT/training/velodyne, and the seed label is generated according to readme into generatecluster_mask/intermediate_results/nusc_bbox_pp_score_fw_20_r0.3.
![image](https://github.com/YurongYou/MODEST/assets/22017747/2a93553d-17f2-4392-80d2-673d12c239a9)
![image](https://github.com/YurongYou/MODEST/assets/22017747/10e3c785-223f-4979-85e3-f4670108505b)

**Comment by Truring** [comment]
Sorry for bothering you. Can you help me with this issue? @YurongYou @katieluo88

**Comment by Harry710887048** [comment]
I'm encountering the same issue. Could it be a problem during the file generation, or is there an error in the coordinate transformation?

**Comment by YurongYou** [comment]
Hi Truring and Harry710887048, thanks for your interest! Looks like it is a file miss match issue, it could be something wrong when setting the file paths. Could you check the following?
* Could you check the number of .txt files generated in the `intermediate_results/nusc_bbox_pp_score_fw_20_r0.3 ` is equal to 3985 (training set size)?
* Could you paint the point cloud with corresponding pp score? (`intermediate_results/nusc_pp_score_fw_30_r0.3/`)
